src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/EditorSettings.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Internal;
+
+namespace Microsoft.CodeAnalysis.Razor.ProjectSystem
+{
+    public abstract class EditorSettings : IEquatable<EditorSettings>
```

Comment:
sealed makes more sense here. We don't anticipate having drastically different editor behavior across platforms